test/integration/net_test.go

```diff
@@ -69,6 +69,13 @@ func TestNetworkPlugins(t *testing.T) {
 					t.Skipf("flannel is not yet compatible with Docker driver: iptables v1.8.3 (legacy): Couldn't load target `CNI-x': No such file or directory")
 				}
 
+				if !DockerDriver() && tc.name == "kubenet" {
+					// CNI is disabled when --network-plugin=kubenet option is passed. See cni.New(..) function
+					// But for containerd/crio CNI has to be configured
+					t.Skipf("Skipping the test as %s container runtimes requires CNI", ContainerRuntime())
+					return
```

Comment:
The `return` is redundant since `t.Skipf` calls `t.SkipNow`

`SkipNow marks the test as having been skipped and stops its execution by calling runtime.Goexit.`